src/app/downloads/table/scrolling-table.component.css

```diff
@@ -0,0 +1,50 @@
+:host {
+  display: block;
+  flex-grow: 1;
+  overflow-x: auto;
+}
+
```

Comment:
This scrolling table layout didn't really change from the original. I just split it off to its own component. It worked with the new design well enough, and I think it's going away, maybe with #283?